banners/desktop/C24_WMDE_Desktop_DE_04/styles/Banner.scss

```diff
@@ -0,0 +1,23 @@
+@use 'src/themes/Treedip/variables/fonts';
```

Comment:
I just noticed that the imports in all these files point to the wrong theme. Can you fix them?